docker-compose.yml

```diff
@@ -748,6 +748,65 @@ services:
   #     - SMARTHOST_HOST=smtp
   #     - SMARTHOST_PORT=587
 
+  xtrabackup:
+    image: percona/percona-xtrabackup:latest
+    container_name: appwrite-xtrabackup
+    command: sleep infinity
+    volumes:
+      - ./backups:/backups:rw
+      - ./varlibmysql:/varlibmysql:rw
```

Comment:
Since the backup uses:
appwrite-mariadb:/var/lib/mysql:r
I can not mount this folder as well for the restore, so only on dev we will write it to a different location.
In production we will not have this line.

Reply:
This needs to be changed.. Reminder for me..

Reply:
@christyjacob4 Do not know how to solve this without creating a new container for xtrabackup Restore

Reply:
Will use the same volume to restore the data.
appwrite-mariadb:/var/lib/mysql:rw